goal_src/engine/sparticle/sparticle.gc

```diff
@@ -678,6 +681,15 @@
 
 (defun set-particle-frame-time ((arg0 int))
   (cond
+    ((= arg0 2)
+     (set-vector!
+      *sp-frame-time*
+      0.000000000000000000000000000000000000011754947
+      2.0
+      0.4
+      0.4
+      )
+     )
```

Comment:
No clue why Naughty Dog is using cases here, but you can turn this whole function into just a formula:
```lisp
(set-vector! *sp-frame-time* (the-as float (logior #x800000 arg0)) (the float arg0) (* 0.2 arg0) (* 0.2 arg0))
```